api/src/index.tsx

```diff
@@ -54,18 +23,37 @@ const PageContent = styled.div`
 `;
 
 function App() {
+  const [darkmode, setDarkmode] = React.useState(false);
+
+  useEffect(() => {
+    const dark = window.localStorage.getItem("darkmode");
```

Comment:
Can you match docosarus' behaviour here?

It sets item `theme` to `dark`, it also defaults for `window.matchMedia("(prefers-color-scheme: dark)")`